services/printers/paperboy/paperboy/printer.py

```diff
@@ -43,7 +65,10 @@ def get_status(self) -> str:
         printer_status = (
             self.printer.get_id() if self.printer else "an unselected printer"
         )
-        return f"You're printing {self.copies} copy(s) to {printer_status}. Modify your options below:"
+        color_status = (
+            "" if not self.printer else " in color" if self.color else " in greyscale"
```

Comment:
Might it make sense to force grayscale if not explicitly printing to a color printer? That way people don't incorrectly assume it will be printed in color.